code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -996,6 +915,14 @@
 	color = "#C0C0C0" //ditto
```

Comment:
should also list that it's an anticonvulsant in the desc now. maybe add to wiki

Reply:
aba9a1c

I added it to the description, I will make wiki changes when the PR is approved, so people arent confused.